### DISCUSSION TITLE
Welcome to sing-land Discussions!

### DISCUSSION THREAD

**Discussion by anonysoul** [net-breaker/sing-land#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by neerro** [comment]
Are you planning to support updated versions of your app? It's very simple and user-friendly. It would be great if you continued to support it.
